test/parse/module/bad-table-invalid-function.txt

```diff
@@ -0,0 +1,10 @@
+;;; ERROR: 1
+(module
+  (table 1 anyfunc)
+  (func $f)
+  (elem (i32.const 0) $f 1))
+(;; STDERR ;;;
+parse/module/bad-table-invalid-function.txt:5:26: function variable out of range (max 1)
+  (elem (i32.const 0) $f 1))
```

Comment:
Hm, good point. Probably better to fix in another PR, because it will affect a lot of tests.